staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -62,6 +69,25 @@ var (
 		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component"},
 	)
 	requestLatencies = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "apiserver_request_latency_seconds",
+			Help: "Response latency distribution in seconds for each verb, group, version, resource, subresource, scope and component.",
+			// Use buckets ranging from 125 ms to 8 seconds.
+			Buckets: prometheus.ExponentialBuckets(0.125, 2.0, 7),
+		},
+		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component"},
+	)
+	requestLatenciesSummary = prometheus.NewSummaryVec(
```

Comment:
Do we want to export summary metrics in the future? I thought we were generally trying to move off of them.

Reply:
I'm not sure whether to remove all these existing summary metrics or convert to histogram.

@brancz @loburm : can you give some suggestion?

Reply:
I believe the summaries are currently in use by performance tests, so we'll need to keep them around for now, but eventually I agree we should just delete them or at least have them be opt in.

Reply:
@brancz, but this PR is adding a new summary metric. How can it already be in use by perf tests?

Reply:
oops I missed that, I meant the existing summary metric (the one that is deprecated) .. my thinking is to just skip the new summary metric as summaries are too expensive here